openapi.yml

```diff
@@ -248,6 +248,10 @@ components:
           type: array
           items:
             $ref: "#/components/schemas/DescriptiveBasicValue"
+    Barcode:
```

Comment:
Is it possible to model the three known types of barcodes separately, then we'd know something of their provenance.  We could have `StandardBarcode: '^36105[0-9]{9}$'`, `BusinessBarcode: '^2050[0-9]{10}$'`, `CatkeyBarcode: '^[0-9]{7}-[0-9]{4}$'`

then:
```yaml
  barcode:
    oneOf:
      - $ref: '#/components/schemas/StandardBarcode'
      - $ref: '#/components/schemas/BusinessBarcode'
      - $ref: '#/components/schemas/CatkeyBarcode'
```

Reply:
I like this.